src/openrct2/ride/coaster/BolligerMabillardTrack.hpp

```diff
@@ -11819,6 +11819,438 @@ void bolliger_mabillard_track_booster(
     paint_util_set_general_support_height(session, height + 32, 0x20);
 }
 
+static void bolliger_mabillard_track_left_twist_down_to_up(
```

Comment:
Use ```inline``` over ```static``` when you implement functions in headers, inline will ensure you will only have a single function instead of a copy per TU, this applies to all the others as well.

Reply:
All the other functions are templates, should they be modified as well?

@karst You didn’t make these template functions, unlike the others, which probably explains why they didn’t work for you.

Reply:
Ideally you want inline for anything you don't want a duplicate of when implemented in a header.

Reply:
Ah gotchu I'll try